driver/src/main/scala/core/protocol/MongoHandler.scala

```diff
@@ -94,6 +94,7 @@ private[reactivemongo] class MongoHandler(
   }
 
   override def exceptionCaught(ctx: ChannelHandlerContext, cause: Throwable) = {
+    println(s"ERROR: $cause")
```

Comment:
There is a `log` just bellow

Reply:
Yes, and I already told you that *if* one *does* enable the trace log level, that the exceptions would be printed. You, however, stated that "I haven't seen such case locally."

This println statement just makes it obvious that an exception is thrown almost for every single message flowing through the decoder.

I don't know what you want... :confused:

Reply:
If that's the reproducer, no need to create a PR, just ping me on a branch. That surprised me in this way.